src/components/common/SwapPrice.stories.tsx

```diff
@@ -19,6 +19,7 @@ const Template: Story<Partial<Props>> = (props) => {
     <SwapPrice
       baseToken={GNO}
       quoteToken={DAI}
+      showBaseToken={true}
       isPriceInverted={isPriceInverted}
       onSwapPrices={(): void => {
```

Comment:
Setting `isPriceInverted` inside Template interferes with Controls. Try clicking `isPriceInverted` in Controls and then :arrows_counterclockwise:, stops working, you essentially make a controlled component into uncontrolled on-the-fly, bad UX.

Stick to one or another -- either remove `onSwapPrices` or disable`isPriceInverted` control

Reply:
I'll disable `isPriceInverted` control